docs/source/model_design/concept_models.rst

```diff
@@ -0,0 +1,245 @@
+..
+  Section title decorators for this document:
+  ==============
+  Document Title
+  ==============
+  Section Level 1
+  ---------------
+  Section Level 2
+  +++++++++++++++
+  Section Level 3
+  ~~~~~~~~~~~~~~~
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+  Section Level 5
+  '''''''''''''''
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+
+================================================
+Vivarium Model Considerations and Best Practices
+================================================
+
+The purpose of this page is to document information that is relevant to designing Vivarium simulation models as a whole, rather than information specific to particular sub-components of Vivarium simulations. This page is arranged in no particular order and topics can be added as they are encountered. The sections may be filled in as simulation science team members investigate the issues in their own projects and can include text from or links to existing documents that may address the topic.
+
+.. contents::
+   :local:
+
+Model Design/Building Topics
+----------------------------
+
+Common random numbers 
++++++++++++++++++++++
+
+.. todo::
+
+	- Between model scenarios
+	- Between model locations
+
+Statistical distributions of uncertainty
+++++++++++++++++++++++++++++++++++++++++
+
+.. todo::
+
+	Intro
+
+Common Statistical Distributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. todo::
+
+	- Uniform
+	- Normal/Truncated normal
+	- Lognormal
+	- Gamma
+	- Beta
+	- Ensemble
+	- Etc.
+
+Common Model Parameters and Their Possible Appropriate Uncertainty Distributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. todo::
+
+	- Relative risk
+	- Mean difference
+	- Proportion
+	- Cost estimate
+	- Etc.
+
+Other Considerations
+~~~~~~~~~~~~~~~~~~~~
+
+.. todo:: 
+
+	- How to handle very asymmetric confidence intervals
+	- How to handle uncertainty in data source(s) rather than statistical uncertainty from a single high quality data source?
+		- Ex: combining multiple estimates from published papers with their own statistical uncertainty
+	- How to handle uncertaity when extrapolating a subnataional estimate to a national estimate?
+	- How to handle uncertainty distribution in the case of joint distributions
```

Comment:
I think this might deserve to be its own section, because we are *always* modeling joint distributions -- if there is more than one random parameter in our model, then they must have a joint distribution.  The "default" is to use the joint distribution where everything is independent, unless we specifically tell the software engineers to couple some of the parameters in some other way.  But I think we should be putting more emphasis on thinking about what joint distribution we are actually using, rather than relegating it to an afterthought.

Reply:
Ah, are you referring to the total effect of the uncertainty intervals for all of the parameters in our model? I would classify this concept as uncertainty in monte carlo analysis, which has its own section currently (at the end!). Is that what you mean or am I misinterpreting?

Reply:
Typically we model parameters, risk factors, etc. to be independent.  As a result, the joint distribution exists, but it's just the product of the marginal distributions, and uncertainty is, obviously, independent.  As such, I don't see added value in interpreting this case as a "joint" distribution, but maybe I'm not looking far enough ahead.
If we have conditionally dependent variables, however, this gets more complicated.  My understanding is that we are just beginning to include things like correlated risks into the simulation models.